app/assets/templates/status-message-location_tpl.jst.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if location}}
   <div class='near-from'>
-    {{ t "publisher.near_from" location=location}}
+    <a href="/posts/{{id}}">{{ t "publisher.near_from" location=location}}</a>
```

Comment:
Not sure this should be an actual link, if anything then to some external map showing the POI. But I'm leaning more towards just styling it as one with `cursor: pointer` and `text-decoration: underline` on the hover state of the parent container.